relay-event-normalization/src/event.rs

```diff
@@ -818,6 +818,16 @@ pub fn normalize_performance_score(
                     );
                 }
                 if should_add_total {
+                    if profile.version.is_some() {
```

Comment:
With this, you don't need the `.unwrap()`:

if let Some(version) = profile.version {